config/locales/en.yml

```diff
@@ -1144,7 +1144,7 @@ en:
       preamble_html: Login with your <strong>%{domain}</strong> credentials. If your account is hosted on a different server, you will not be able to log in here.
       title: Login to %{domain}
     sign_up:
-      manual_review: Sign-ups on %{domain} go through manual review by our moderators. To help us process your registration, write a bit about yourself and why you want an account on %{domain}.
```

Comment:
I've placed this removed text as the default message. It's not fantastic that it's not localisable anymore, but I don't see a way to localise settings at this time.

Reply:
That is a fairly big regression, I think it would make more sense for this to stay, and be used when the custom message is not set.

Reply:
Yeah, I think we probably want some combination of what I have now and the existing localisations for the message. Perhaps `manual_review` remains, but we add a `manual_review_with_message` which is just the first sentence: "Sign-ups on %{domain} go through manual review by our moderators."

At the same time, we probably want to think harder about how to localise certain settings, e.g., the about text, rules, this message, etc.